conan_provider.cmake

```diff
@@ -451,8 +451,46 @@ function(conan_install)
     # success
     set_property(GLOBAL PROPERTY CONAN_INSTALL_SUCCESS TRUE)
 
+
+    conan_set_program_path()
+
+    list(PREPEND CMAKE_PROGRAM_PATH ${_CONAN_PROGRAM_PATH})
+    set(CMAKE_PROGRAM_PATH ${CMAKE_PROGRAM_PATH} PARENT_SCOPE)
+
 endfunction()
 
+macro(conan_set_program_path)
```

Comment:
Maybe it would be a bit cleaner to do a function and "return" the variable to be used later